lib/logflare/cache_buster.ex

```diff
@@ -15,13 +15,47 @@ defmodule Logflare.CacheBuster do
   end
 
   def init(state) do
+    Logger.put_process_level(self(), :error)
+
     Cainophile.Adapters.Postgres.subscribe(Logflare.PgPublisher, self())
     {:ok, state}
   end
 
+  @doc """
+  Sets the Logger level for this process. It's started with level :error.
+
+  To devug wal records set process to level :info and each transaction will be logged.
+  """
+
+  @spec set_log_level(
+          :alert
+          | :all
+          | :critical
+          | :debug
+          | :emergency
+          | :error
+          | :info
+          | :none
+          | :notice
+          | :warn
+          | :warning
+        ) :: :ok
+  def set_log_level(level) when is_atom(level) do
+    Process.whereis(__MODULE__)
+    |> GenServer.call({:put_level, level})
```

Comment:
```suggestion
    GenServer.call(__MODULE__, {:put_level, level})
```
Module name is already registered as process name, no need to find pid.